### PR TITLE
Moving stripe samples to it's own menu

### LINKED ISSUE #249: Stripe Panel Updates
* Move the samples out into its own view. Panel should be: Events, Logs, Samples, Quick Links, Help and Feedback
* Auto-collapse the quick links and help and feedback section. 
* Remove "Find code samples" from the help and feedback link. This belongs in the context of the Stripe Samples workflow.

### PR DESCRIPTION
**Pull request by ctrudeau-stripe**
### Summary 

- Move the samples out into its own view. Panel should be: Events, Logs, Samples, Quick Links, Help and Feedback
- Auto-collapse the quick links and help and feedback section.
- Remove "Find code samples" from the help and feedback link. This belongs in the context of the Stripe Samples workflow.

### Testing 

Manually tested:

Screen Shot 2021-06-07 at 5 30 40 PM
Screen Shot 2021-06-07 at 5 30 36 PM

fixes #249 
r? @vcheung-stripe

### CODE REVIEW

File: src/stripeHelpView.ts [hunk 1]
```diff
@@ -12,13 +12,6 @@ export class StripeHelpViewProvider extends StripeTreeViewDataProvider {
     });
     items.push(docsItem);
 
-    const samplesItem = new StripeTreeItem('Find code samples', {
```
Comment (vcheung-stripe) [inline src/stripeHelpView.ts:15]: Can we put `stripe.openSamples` inside the context menu of **Start with a Stripe Sample**? For example, `openDashboardEvent` is available when you right-click an event item when you start streaming events.

**Review by vcheung-stripe — CHANGES_REQUESTED** [review]
Nice! Just had a comment about putting the stripe samples link in the context menu.

File: src/stripeHelpView.ts (hunk 1 above)
Comment (ctrudeau-stripe) [reply src/stripeHelpView.ts:15]: Sounds good to me

File: src/stripeSamplesView.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    });
+
+    items.push(samplesItem);
+
+    const findSamplesItem = new StripeTreeItem('Find code samples', {
+      commandString: 'openSamples',
+      iconPath: new ThemeIcon('code'),
+      tooltip: 'Sample integrations built by Stripe',
+    });
+    items.push(findSamplesItem);
```
Comment (vcheung-stripe) [inline src/stripeSamplesView.ts:22]: Oh apologies if I wasn't clear! The `Find code samples` link should be available when you right-click the `Start with a Stripe Sample` link. It involves
- adding a `contextValue` to the `samplesItem`
- registering `stripe.openSamples` in the package.json similar to how `stripe.openDashboardEvent` is registered.

`stripe.openDashboardEvent` should be a good example of how to do this!

Comment (Trudeaucj) [reply src/stripeSamplesView.ts:22]: Ah had no idea this menu existed!

**Review by vcheung-stripe — APPROVED** [review]
LGTM

**Pull request merged**